Sources/NIOWebSocket/WebSocketFrameDecoder.swift

```diff
@@ -221,7 +221,7 @@ public final class WebSocketFrameDecoder: ByteToMessageDecoder {
     public var cumulationBuffer: ByteBuffer? = nil
 
     /// The maximum frame size the decoder is willing to tolerate from the remote peer.
-    private let maxFrameSize: Int
+    /* private but tests */ let maxFrameSize: Int
```

Comment:
I think it make even more sense to test behavior (e.g., ability _and_ inability to transfer more than the limit when the limit is appropriately changed) rather than checking whether we set the `maxFrameSize`. It'll have a nice side effect of eliminating this comment.

Reply:
@vlm I will add more tests as a followup to test max frame size handling. But I think this is ok to test what the change did.